modules/api/src/main/java/org/apache/ignite/lang/ErrorGroups.java

```diff
@@ -577,6 +577,13 @@ public static class Compute {
 
         /** Specified node is not found in the cluster. */
         public static final int NODE_NOT_FOUND_ERR = COMPUTE_ERR_GROUP.registerErrorCode((short) 14);
+
+        /**
+         * Incompatible types for argument/result in compute job.
+         * For example, the one has defined a marshaller for Type A in the compute job
+         * but on the client side they have passed Type B.
+         */
+        public static final int TYPE_CHECK_MARSHALLING_ERR = COMPUTE_ERR_GROUP.registerErrorCode((short) 15);
```

Comment:
```suggestion
        public static final int MARSHALLING_TYPE_MISMATCH_ERR = COMPUTE_ERR_GROUP.registerErrorCode((short) 15);
```

(subject comes first in the error name)